ChefExtensionHandler/bin/chef-install.psm1

```diff
@@ -72,7 +74,21 @@ function Install-ChefClient {
   $env:Path += ";C:\\opscode\\chef\\bin;C:\\opscode\\chef\\embedded\\bin"
 
   Install-AzureChefExtensionGem $chefExtensionRoot
+}
 
+function Download-ChefClient {
+  $remoteUrl = "http://www.chef.io/chef/download?p=windows&pv=2012&m=x86_64&v=latest&prerelease=false"
```

Comment:
I think next step will be take a `chef-client` version from user and form this url? Please put todo note here!